Calling/ClientApp/src/components/MediaGallery.tsx

```diff
@@ -25,31 +25,20 @@ export default (props: MediaGalleryProps): JSX.Element => {
   const [gridCol, setGridCol] = useState(1);
   const [gridRow, setGridRow] = useState(1);
 
-  if (Constants.DOMINANT_PARTICIPANTS_COUNT < 1 || Constants.DOMINANT_PARTICIPANTS_COUNT > 8) {
-    console.error('Please use a value for dominante participants between 1 <= x <= 8');
-  }
-
   // we only are going to support up to a 3x3 grid for today (1 local + 8 remote)
-  const rows = [1, 1, 2, 2, 2, 2, 3, 3, 3];
-  const cols = [1, 2, 2, 2, 3, 3, 3, 3, 3];
-
-  const calculateNumberOfRows = React.useCallback((participants, maxStreamsToRender) => {
-    const length = Math.min(participants.length, maxStreamsToRender);
-    if (length - 1 >= rows.length) {
-      return 3;
-    }
+  // remote participants 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15
```

Comment:
Could you add some description to this comment. I didn't understand it after a couple readings.